source/simulator/helper_functions.cc

```diff
@@ -1112,7 +1112,7 @@ namespace aspect
   double
   Simulator<dim>::compute_initial_stokes_residual()
   {
-    LinearAlgebra::BlockVector remap (introspection.index_sets.stokes_partitioning, mpi_communicator);
+    LinearAlgebra::BlockVector zero_vel_sol (introspection.index_sets.stokes_partitioning, mpi_communicator);
```

Comment:
I'm not fond of this variable name either. Could you spell it out completely? It's also not clear to me what the "zero" part is supposed mean. Could you maybe add a comment what the variable is supposed to be in the first place?